L1Trigger/L1THGCal/interface/be_algorithms/HGCalTriggerCellCalibration.h

```diff
@@ -16,21 +16,29 @@
 
 class HGCalTriggerCellCalibration{
 
+private:
+    // FIXME: currently there is no access to the HGCal DDDConstants
```

Comment:
It will probably be done after the TDR, together with larger structural changes of the code